### ISSUE TITLE
Timelapse smooth travel always present in slicing - no chance to deactivate it

### ISSUE THREAD

**Issue by HorstBaumann** [bambulab/BambuStudio#3161]
### Bambu Studio Version

1.8.2.56

### Where is the application from?

Bambu Lab Official website

### OS version

Windows 10

### Additional system information

core x7

### Printer

A1 mini

### How to reproduce

any object i slice, gets the "smooth timelapse" typical paths to the back left corner.
Even when the timelaps setting is on "traditional" or timelapse is deactivated.
bambu studio seems to ignore these settings.

![grafik](https://github.com/bambulab/BambuStudio/assets/11771888/df9b70dd-4126-4963-82a8-688e6ea805d7)

### Actual results

the printed object has ugly filament loops and the printing time increases

### Expected results

a clean print without unnecessary travels

### Project file & Debug log uploads

the problem is not related to a certain project
[Bowling Trophy.zip](https://github.com/bambulab/BambuStudio/files/13706484/Bowling.Trophy.zip)

### Checklist of files to include

- [ ] Log file
- [X] Project file

**Comment by bachrach44** [comment]
The A1 is always going to have to move the print head and bed for reasons explained here: https://wiki.bambulab.com/en/software/bambu-studio/Timelapse

However, it would be nice if we could just disable timelapse entirely. It was fun at first, but after a while it's just a waste of time and filament.

**Comment by DConcord** [comment]
+1 - I've had occasional luck when toggling Others > Print Sequence > By Object because that mode is not compatible with timelapse. But toggling back doesn't always keep the mystery/hidden timelapse setting disabled

**Comment by loohney** [comment]
+1 - We need an option to entirely disable the timelapse feature.

**Comment by zhimin-zeng-bambulab** [comment]
![image](https://github.com/bambulab/BambuStudio/assets/120697589/e0dc2b2e-04cc-46ac-847e-3b00d14cc5ae)
If you choose traditional timelapse and do not check timelapse when sending print job, this GCode will be skipped when printing.

**Comment by DConcord** [comment]
> ![image](https://private-user-images.githubusercontent.com/120697589/293935806-e0dc2b2e-04cc-46ac-847e-3b00d14cc5ae.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQyOTIxMjksIm5iZiI6MTcwNDI5MTgyOSwicGF0aCI6Ii8xMjA2OTc1ODkvMjkzOTM1ODA2LWUwZGMyYjJlLTA0Y2MtNDZhYy04NDdlLTNiMDBkMTRjYzVhZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEwM1QxNDIzNDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wMDIyNjQwNmM5MmIxYjFhZDJiOWYwMDU3YzA3MzYyYjI3MzBiMDk4ZDE3NTE5Yzg1YzllYjg4NmVmM2Q4YmU5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.IBszUJgl2jP7wsKcTQ3zH6I-ycwMVycukkTjj9rK-Bs) If you choose traditional timelapse and do not check timelapse when sending print job, this GCode will be skipped when printing.

Right, but the estimated time is still thrown off. Especially on bigger prints

**Comment by SaltWei** [comment]
> Right, but the estimated time is still thrown off. Especially on bigger prints

This time error is very small and only several minutes for a 16 hours printing.

**Comment by Hammerfest** [comment]
v1.9.2 and its still an issue

such a waste of time, just give us the option to turn it off... such a hassle

**Comment by zhimin-zeng-bambulab** [comment]
@Hammerfest  There is a timelapse  switch when sending for printing. If you think it is a waste of time, uncheck timelapse and the gcode to the trash will not be executed.

**Comment by Hammerfest** [comment]
I do, but I think your missing the point, on the preview, its displays the movement regardless of option, if your trying to optimize line travel this flys in the face of and defeats that ability

**Comment by lightspeedlucas** [comment]
+1 - Note that as a workaround, it seems you can fully erase `Time lapse G-code` in the `Machine gcode` tab in your printer's profile and the preview/timing will then work as we expect.

**Comment by hutje** [comment]
+1, just also give the option to make a timelapse without the travel movements. Octoprint does provide it and it is just ok for most people. Now it adds 20% extra total time for the travel movements.

**Comment by Edya7659** [comment]
Couldn't you just add "Disable" to the Special Mode/Timelapse menu, and then the movement of the print head to the shooting position would not be calculated (disappeared)? Yes, in that case, there would be NO possibility of enabling Timelapse shooting when sending the model to print. But, on the other hand, it would be possible to get more accurate data on the model's printing time.

![Timelapse menu](https://github.com/user-attachments/assets/2b038f77-f555-4f2f-b717-948198ef8ada)

**Comment by rrodriguez-medicaljustice** [comment]
I just recently bought the A1 and I was wondering about this.

Please just add an option to include or exclude time lapse travel moves, having the time lapse travel moves show up always sis very confusing. 
******I have just spent a few hours trying to figure out why my print was going to take a long time, and figuring out why all those travel lines.****

**Comment by joojoooo** [comment]
Simply add a disable option? 2 year later and that's still an issue, what's taking so long? Bambu devs don't thing it's an issue? Is it on purpose?

Total time is wrong and all the travel moves timelapse adds in the preview make it impossible to see the real moves that are actually gonna happen, sometimes you need to see where the nozzle will move, cause maybe it's dragging on a surface or causing stringing, timelapse "ghost moves" makes it harder to see the real travel.

Also, that would make the GCODE smaller, meaning faster upload and less waste of storage. No one is using timelapse regularly, it should be disabled BY DEFAULT, not enabled by default !

**Comment by Tyson-08L** [comment]
> +1 - I've had occasional luck when toggling Others > Print Sequence > By Object because that mode is not compatible with timelapse (and disables it somehow?). But toggling back doesn't always keep the mystery/hidden timelapse setting disabled

THIS SOLVED IT FOR ME!!!!! THANK YOU SO MUCH!!!
